### PR TITLE
add documentation structure and guidelines

### PR DESCRIPTION
**Pull request by samccann**

### CODE REVIEW

File: docs/docs_toolkit.md [hunk 1]
```diff
@@ -1 +1,179 @@
-# Documentation tookit
\ No newline at end of file
+# Documenting your project
```
Comment (oraNod) [inline docs/docs_toolkit.md:1]:
```suggestion
# Documenting Ansible projects
```

File: docs/docs_toolkit.md [hunk 2]
```diff
@@ -1 +1,179 @@
...
+
+This section helps you add documentation to your Ansible project or improve your existing documentation.
+The onboarding [project-template](https://github.com/ansible-community/project-template)
+in GitHub gives you the scaffolding files to start your documentation effort.
```
Comment (oraNod) [inline docs/docs_toolkit.md:5]:
```suggestion
provides a set of base files to quickly create a new documentation set. 
```

File: docs/docs_toolkit.md [hunk 3]
```diff
@@ -1 +1,179 @@
-# Documentation tookit
\ No newline at end of file
+# Documenting your project
+
+This section helps you add documentation to your Ansible project or improve your existing documentation.
+The onboarding [project-template](https://github.com/ansible-community/project-template)
+in GitHub gives you the scaffolding files to start your documentation effort. 
+
+## Creating new documentation projects
```
Comment (oraNod) [inline docs/docs_toolkit.md:7]:
```suggestion
## Creating new documentation sets
```

Comment (oraNod) [reply docs/docs_toolkit.md:7]: I think we should say either "documentation set" or maybe "docsite" here. For me "documentation project" sounds like a standalone thing dedicated only to docs.

File: docs/docs_toolkit.md [hunk 4]
```diff
@@ -1 +1,179 @@
...
+
+Ansible projects starting a new documentation effort should follow these guidelines:
```
Comment (oraNod) [inline docs/docs_toolkit.md:9]:
```suggestion
Documentation guidelines for new Ansible projects:
```

File: docs/docs_toolkit.md [hunk 5]
```diff
@@ -1 +1,179 @@
...
+
+* Markdown is simple, ubiquitous, and familiar to most contributors. For these reasons
+ the Ansible community team recommends markdown as the format for documentation.
+* [mkdocs](https://www.mkdocs.org/getting-started/) is the recommended tool for building 
+the documentation from markdown.
```
Comment (oraNod) [inline docs/docs_toolkit.md:14]:
```suggestion
* [mkdocs](https://www.mkdocs.org/getting-started/) is the recommended tool for building 
the doc site from markdown source.
```

File: docs/docs_toolkit.md [hunk 6]
```diff
@@ -1 +1,179 @@
...
+* [ReadtheDocs](https://readthedocs.org/) is the recommended place to publish documentation
+ for Ansible projects.
+* The [Ansible style guide](https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html) 
+is available to help ensure your project is well-written and follows technical documentation guidelines.
+
+Follow these steps to set up your documentation if you do not use the GitHub project-template:
```
Comment (oraNod) [inline docs/docs_toolkit.md:20]:
```suggestion
If you do not use the project-template, follow these steps to set up your documentation:
```

File: docs/docs_toolkit.md [hunk 7]
```diff
@@ -1 +1,179 @@
...
+
+1. Create a ``/docs`` directory in your project repository.
```
Comment (oraNod) [inline docs/docs_toolkit.md:22]:
```suggestion
1. Create a ``/docs`` directory at the base of your project repository. 
```

File: docs/docs_toolkit.md [hunk 8]
```diff
@@ -1 +1,179 @@
...
+ for Ansible projects.
+* The [Ansible style guide](https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html) 
+is available to help ensure your project is well-written and follows technical documentation guidelines.
+
+Follow these steps to set up your documentation if you do not use the GitHub project-template:
+
+1. Create a ``/docs`` directory in your project repository. 
+2. Follow the proposed [documentation structure](https://github.com/ansible-community/project-template/tree/main/docs)
+ from the project-template to ensure you include all the important guides common
+to all projects.
```
Comment (oraNod) [inline docs/docs_toolkit.md:25]:
```suggestion
2. Create the proposed [documentation structure](https://github.com/ansible-community/project-template/tree/main/docs) from the project-template to include the guides that are common to Ansible projects.
```

File: docs/docs_toolkit.md [hunk 9]
```diff
@@ -1 +1,179 @@
...
+3. Add a [`mkdocs.yml`](https://github.com/ansible-community/project-template/blob/main/mkdocs.yml) to
+ your project root directory and modify the `nav` section to match your proposed navigation/set of guides.
```
Comment (oraNod) [inline docs/docs_toolkit.md:27]:
```suggestion
3. Set up your documentation build in the [`mkdocs.yml`](https://github.com/ansible-community/project-template/blob/main/mkdocs.yml) configuration file so the `nav` section includes all your project guides.
```

File: docs/docs_toolkit.md [hunk 10]
```diff
@@ -1 +1,179 @@
...
+4. Create [``.readthedocs.yml``](https://github.com/ansible-community/project-template/blob/main/.readthedocs.yaml) in your project root directory.
```
Comment (oraNod) [inline docs/docs_toolkit.md:28]:
```suggestion
4. Add the Read The Docs configuration file, [``.readthedocs.yml``](https://github.com/ansible-community/project-template/blob/main/.readthedocs.yaml), to your project's base.
```

File: docs/docs_toolkit.md [hunk 11]
```diff
@@ -1 +1,179 @@
...
+5. Coordinate with the [Ansible community team](https://docs.ansible.com/ansible/latest/community/communication.html#forum) to set up automated publishing and to have your documentation added to the Ansible ecosystem page.
```
Comment (oraNod) [inline docs/docs_toolkit.md:29]:
```suggestion
5. Ask the [Ansible community team](https://docs.ansible.com/ansible/latest/community/communication.html#forum) to help set up server-side configuration on Read The Docs and add your project to the Ansible namespace.
```

Comment (oraNod) [reply docs/docs_toolkit.md:29]: This last bit seems a little risky to me. Does this onboarding apply only to projects in the Ansible org?

Comment (samccann) [reply docs/docs_toolkit.md:29]: This is meant for 'official' ecosystem projects only. Not sure what namespace that is? While anyone can use these guidelines for any project they want, it is meant for projects that will be part of the official Ansible community projects.

**Pull request merged**